examples/datasource-http-backend/README.md

```diff
@@ -1,10 +1,18 @@
 # Grafana Data Source Plugin Template
 
-This example queries data from an HTTP API.
+This example queries data from an HTTP API, which returns data in JSON format and then it's converted to data frames.
 
-This differs from datasource-http, because the data fetching happens on the backend.
+This differs from the datasource-http example, because the data fetching happens on the backend.
```

Comment:
Maybe something along these lines?

```suggestion
This differs from the datasource-http example, because the data fetching happens in the plugin backend rather than going through Grafana's datasource HTTP proxy.
```